### PR TITLE
feat: added metadata_properties inside the __repr__ method in both remote and local folder

### PR DESCRIPTION
**Pull request by subhajit20**
…and local folder

# Argilla Community Growers
Added metadata_properties inside the __repr__ method in both remote and local folder
Issue - #4190 

Thanks for your contribution! As part of our Community Growers initiative 🌱, we're donating Justdiggit bunds in your name to reforest sub-Saharan Africa. To claim your Community Growers certificate, please contact David Berenstein in our Slack community or fill in this form https://tally.so/r/n9XrxK once your PR has been merged.

# Pull Request Templates

Please go the the `Preview` tab and select the appropriate sub-template:

* [🐞-bug](?expand=1&template=bug.md)
* [📚-documentation](?expand=1&template=docs.md)
* [🆕-features](?expand=1&template=features.md)

# Generic Pull Request Template

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #<issue_number>

**Type of change**

(Please delete options that are not relevant. Remember to title the PR according to the type of change)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (change restructuring the codebase without changing functionality)
- [ ] Improvement (change adding some improvement to an existing functionality)
- [ ] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes. And ideally, reference `tests`)

- [ ] Test A
- [ ] Test B

**Checklist**

- [ ] I added relevant documentation
- [ ] follows the style guidelines of this project
- [ ] I did a self-review of my code
- [ ] I made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
- [ ] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: src/argilla/client/feedback/dataset/remote/dataset.py [hunk 1]
```diff
@@ -538,6 +538,7 @@ def __repr__(self) -> str:
             + textwrap.indent(f"\nfields={self.fields}", indent)
             + textwrap.indent(f"\nquestions={self.questions}", indent)
             + textwrap.indent(f"\nguidelines={self.guidelines}", indent)
+            + textwrap.indent(f"\nmetadata_properties={self.metadata_properties}", indent)
             + ")"
```
Comment (davidberenstein1957) [inline src/argilla/client/feedback/dataset/remote/dataset.py]:
```suggestion
            + "\n)"
```

File: src/argilla/client/feedback/dataset/local/dataset.py [hunk 2]
```diff
@@ -214,7 +214,8 @@ def __repr__(self) -> str:
         return (
             "FeedbackDataset("
             + textwrap.indent(
```
Comment (davidberenstein1957) [inline src/argilla/client/feedback/dataset/local/dataset.py]: could you align this with the formatting used for the `RemoteFeedbackDataset

**Comment by davidberenstein1957** [comment]
@subhajit20, lastly, could you update the `changelog.md`?

**Comment by subhajit20** [comment]
> @subhajit20, lastly, could you update the `changelog.md`?

done

**Review by davidberenstein1957 — APPROVED** [review]

**Comment by davidberenstein1957** [comment]
@subhajit20, thanks again for this PR! Just to double check, did you fill out our contributor form? https://tally.so/r/n9XrxK

**Comment by subhajit20** [comment]
> @subhajit20, thanks again for this PR! Just to double check, did you fill out our contributor form? https://tally.so/r/n9XrxK

yeah done that.
Thank You

**Pull request merged**